.cirrus.yml

```diff
@@ -31,7 +31,7 @@ env:
     PRIOR_FEDORA_NAME: "fedora-35"
     UBUNTU_NAME: "ubuntu-2110"
 
-    IMAGE_SUFFIX: "c4955393725038592"
+    IMAGE_SUFFIX: "c6211193021923328"
```

Comment:
I'm guessing this was accidental: it's not mentioned in the commit message, and CI is failing. I'll take another look after this is fixed.

Reply:
Ah, okay; then when you do your final commit, you may want to update the commit message to specify that. Otherwise there's no indication of why the underlying images changed, and that's usually a big change that warrants a big notice.

Reply:
Ya, I agree.  I did mention "Update CI VM images to those which no-longer react to the $DEBUG env. var.".  Hmm, maybe  I should mention the "automation script library" explicitly.

Reply:
Message updated.